README.md

```diff
@@ -264,6 +267,37 @@ Each organization/tenant is associated to a different database.
 
 [Top](#top)
 
+###<a name="section3.4"></a>OrionMongoSink persistence
```

Comment:
Since this information is already available at https://github.com/telefonicaid/fiware-cygnus/blob/feature/363_OrionMongoSink/doc/design/OrionMongoSink.md and it is also reference in the included text, I would suggest to try to explain what can be inferred from the "shell screenshot" with simple words :) Even for me who knows what's going on there was (kind of) difficult to interpret what was included in that "shell screenshot" ;) Just as a suggestion... :)